ad_hoc/español/AnaColon/resumen_Lesser_1997.md

```diff
@@ -0,0 +1,25 @@
+El blanqueamiento de arrecifes de coral es la explosión masiva de dinoflagelados o el perdimiento de los pigmentos fotosintéticos en las zooxantelas. Blanqueamiento ocurre cuando la temperatura sube (lo más mínimo de 1-2 °C) de la temperatura media más alta, radiación UV, cambios en la salinidad, sedimentación pesada, y exposición a contaminantes. 
```

Comment:
```suggestion
El blanqueamiento de arrecifes de coral es la expulsión de dinoflagelados o el perdimiento de sus pigmentos fotosintéticos. Blanqueamiento ocurre cuando la temperatura sube (lo más mínimo de 1-2 °C) de la temperatura media más alta, radiación UV, cambios en la salinidad, sedimentación pesada, y exposición a contaminantes. 
```

Reply:
does the pronoun 'sus' clearly refer to the pigments of the dinos or is it ambiguous whether it refers to pigments in the corals themselves, when written this way?